examples/tv-casting-app/darwin/TvCasting/TvCasting/TvCastingApp.swift

```diff
@@ -38,6 +38,12 @@ struct TvCastingApp: App {
                         }
                         appParameters.rotatingDeviceIdUniqueId = Data(rotatingDeviceIdUniqueId)
                         
+                        let onboardingParameters: OnboardingPayload = OnboardingPayload()
+                        onboardingParameters.setupPasscode = 20202021
```

Comment:
are these defaults?

Reply:
Yes. And this is example code in the iOS tv-casting-app, not part of the casting framework.